frontend/src/components/difficultySelect/DifficultySelect.jsx

```diff
@@ -6,13 +6,17 @@ import { useContext, useState } from 'react';
 import { ChessContext } from '../../pages/ChessPage/ChessContext';
 
 export const DifficultySelect = () => {
+
   const { config, setConfigValue } = useContext(ChessContext);
   const [difficulty, setDifficulty] = useState(config.SKILL_LEVEL);
 
   const handleChange = (event) => {
     const newDifficulty = parseInt(event.target.value, 10);
     setDifficulty(newDifficulty);
+
+
     setConfigValue('SKILL_LEVEL', newDifficulty);
+
   };
```

Comment:
Remember to use the `Prettier` autoformatter when working. I recommend turning on format on save if you haven't already